custodia.spec

```diff
@@ -0,0 +1,263 @@
+tox --sitepackages -e $TOXENV -- --skip-servertests
+%endif
+
+
+%install
+mkdir -p %{buildroot}/%{_sbindir}
+mkdir -p %{buildroot}/%{_mandir}/man7
+mkdir -p %{buildroot}/%{_defaultdocdir}/custodia
+mkdir -p %{buildroot}/%{_defaultdocdir}/custodia/examples
+mkdir -p %{buildroot}/%{_sysconfdir}/custodia
+mkdir -p %{buildroot}/%{_unitdir}
+mkdir -p %{buildroot}/%{_tmpfilesdir}
+mkdir -p %{buildroot}/%{_localstatedir}/lib/custodia
+mkdir -p %{buildroot}/%{_localstatedir}/log/custodia
+
+%{__python2} setup.py install --skip-build --root %{buildroot}
+mv %{buildroot}/%{_bindir}/custodia %{buildroot}/%{_sbindir}/custodia
+cp %{buildroot}/%{_sbindir}/custodia %{buildroot}/%{_sbindir}/custodia-2
+cp %{buildroot}/%{_bindir}/custodia-cli %{buildroot}/%{_bindir}/custodia-cli-2
+install -m 644 -t "%{buildroot}/%{_mandir}/man7" man/custodia.7
+install -m 644 -t "%{buildroot}/%{_defaultdocdir}/custodia" README API.md
+install -m 644 -t "%{buildroot}/%{_defaultdocdir}/custodia/examples" custodia.conf
+install -m 600 %{SOURCE2} %{buildroot}%{_sysconfdir}/custodia
+install -m 644 %{SOURCE3} %{buildroot}%{_unitdir}
+install -m 644 %{SOURCE4} %{buildroot}%{_unitdir}
+install -m 644 %{SOURCE5} %{buildroot}%{_tmpfilesdir}/custodia.conf
+
+%if 0%{?with_python3}
+# overrides /usr/bin/custodia-cli and /usr/sbin/custodia with Python 3 shebang
+%{__python3} setup.py install --skip-build --root %{buildroot}
+mv %{buildroot}/%{_bindir}/custodia %{buildroot}/%{_sbindir}/custodia
+cp %{buildroot}/%{_sbindir}/custodia %{buildroot}/%{_sbindir}/custodia-3
+cp %{buildroot}/%{_bindir}/custodia-cli %{buildroot}/%{_bindir}/custodia-cli-3
+%endif
+
+
+%files
+%doc README API.md
+%doc %{_defaultdocdir}/custodia/examples/custodia.conf
+%license LICENSE
+%{_mandir}/man7/custodia*
+%{_sbindir}/custodia
+%{_bindir}/custodia-cli
+%dir %attr(0700,root,root) %{_sysconfdir}/custodia
+%config(noreplace) %attr(600,root,root) %{_sysconfdir}/custodia/custodia.conf
+%attr(644,root,root)  %{_unitdir}/custodia.socket
+%attr(644,root,root)  %{_unitdir}/custodia.service
+%dir %attr(0700,root,root) %{_localstatedir}/lib/custodia
+%dir %attr(0700,root,root) %{_localstatedir}/log/custodia
+%{_tmpfilesdir}/custodia.conf
+
+%files -n python2-custodia
+%license LICENSE
+%exclude %{python2_sitelib}/custodia/store/etcdstore.py*
+%{python2_sitelib}/*
+%{_sbindir}/custodia-2
+%{_bindir}/custodia-cli-2
+
+%files -n python2-custodia-extra
+%license LICENSE
+%{python2_sitelib}/custodia/store/etcdstore.py*
+
+%if 0%{?with_python3}
+%files -n python3-custodia
+%license LICENSE
+%exclude %{python3_sitelib}/custodia/store/etcdstore.py
+%exclude %{python3_sitelib}/custodia/store/__pycache__/etcdstore.*
+%{python3_sitelib}/*
+%{_sbindir}/custodia-3
+%{_bindir}/custodia-cli-3
+
+%files -n python3-custodia-extra
+%license LICENSE
+%{python3_sitelib}/custodia/store/etcdstore.py
+%{python3_sitelib}/custodia/store/__pycache__/etcdstore.*
+%endif
+
+
+%changelog
```

Comment:
I do not think we want to import fedora's changelog in the upstream repo, let's drop it.